src/hatch/python/resolve.py

```diff
@@ -178,12 +178,24 @@ def get_compatible_distributions() -> dict[str, Distribution]:
     return distributions
 
 
+def _guess_linux_variant() -> str:
+    with open('/proc/cpuinfo', encoding='ascii') as infh:
```

Comment:
After you are content with the six let's please gate this behind `sys.platform not in {'darwin', 'win32'}`

Reply:
should be the case already, see line 199 where it's called... but we can add an extra check to the function itself if you like

Reply:
I wasn't reading closely enough, thanks!